static/app/views/projectInstall/platformOrIntegration.tsx

```diff
@@ -20,7 +21,9 @@ function PlatformOrIntegration(props: Props) {
   }
   return (
     <OnboardingContextProvider>
-      <ProjectInstallPlatform {...props} />
+      <GettingStarted>
```

Comment:
since the overview is no longer being used - it is being deleted in this PR - we had to add this wrapper, adding spacing to the page.